arcsi/model/user.py

```diff
@@ -27,3 +28,20 @@ class User(db.Model, UserMixin):
 
     def __repr__(self):
         return "<Host {}>".format(self.name)
+
+class User2(db.Model, SQLAlchemyUserMixin):
+    """
+    A generic user model that might be used by an app powered by flask-praetorian.
+    Extends the SQLAlchemyUserMixin so that required methods and properties are already
+    well defined
+    """
+
+    __tablename__ = "users2"
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.Text, unique=True)
+    hashed_password = db.Column(db.Text)
+    roles = db.Column(db.Text)
+    is_active = db.Column(db.Boolean, default=True, server_default="true")
+
+    def is_valid(self):
```

Comment:
Small nitpick, sorry but I am curious:

Did you choose `is_valid` because `is_active` is an existing class method you don't want to mess with or just by chance?

If just by chance could I ask you to use `is_active`? I have experienced many times that different method names to argument names (especially if it's a one-line getter) can create confusion. Which is the field name, which is the method, `valid` or `active`... are they equivalent? What if we want to have a separate `valid` property later on, etc.

Reply:
As I remember I just copied this lines from the flask-praetorian's example from [here](https://github.com/dusktreader/flask-praetorian/blob/master/example/basic_with_sqlalchemy_mixin.py)